.github/workflows/ci.yml

```
@@ -2,76 +2,50 @@ name: CI
 on: [push]
 
 jobs:
-  pre-commit-checks:
-    name: Linux - pre-commit checks - Python 3.11
-    timeout-minutes: 30
-    runs-on: ubuntu-latest
-    env:
-      PRE_COMMIT_USE_MICROMAMBA: 1
-    steps:
-      - name: Checkout branch
-        uses: actions/checkout@v4
-      - name: Set up micromamba
-        uses: mamba-org/setup-micromamba@f8b8a1e23a26f60a44c853292711bacfd3eac822
-      - name: Add micromamba to GITHUB_PATH
-        run: echo "${HOME}/micromamba-bin" >> "$GITHUB_PATH"
-      - name: Install Python 3.11
-        uses: actions/setup-python@v5
-        with:
-          python-version: "3.11"
-      - name: Run pre-commit checks
-        uses: pre-commit/action@v3.0.1
+  # pre-commit-checks:
+  #   name: Linux - pre-commit checks - Python 3.11
+  #   timeout-minutes: 30
+  #   runs-on: ubuntu-latest
+  #   env:
+  #     PRE_COMMIT_USE_MICROMAMBA: 1
+  #   steps:
+  #     - name: Checkout branch
+  #       uses: actions/checkout@v4
+  #     - name: Set up micromamba
+  #       uses: mamba-org/setup-micromamba@f8b8a1e23a26f60a44c853292711bacfd3eac822
+  #     - name: Add micromamba to GITHUB_PATH
+  #       run: echo "${HOME}/micromamba-bin" >> "$GITHUB_PATH"
+  #     - name: Install Python 3.11
+  #       uses: actions/setup-python@v5
+  #       with:
+  #         python-version: "3.11"
+  #     - name: Run pre-commit checks
+  #       uses: pre-commit/action@v3.0.1
 
   unit-tests:
-    name: Tests - ${{ matrix.os }} - Py${{ matrix.python-version }}
+    name: Tests - ${{ matrix.os }} - ${{ matrix.environment }}
```

```suggestion
    name: Tests
```

This would render the name to be 

`Tests (ubuntu-latest, py39)`